yang/frr-isisd.yang

```diff
@@ -605,6 +605,26 @@ module frr-isisd {
           "IPv6 destination-source topology.";
       }
     }
+
+    container mpls {
+      description
+	"Configuration of MPLS parameters";
+      leaf ldp-sync {
+        type boolean;
+        default "true";
+        description
+          "Enable MPLS LDP-Sync functionality on this circuit.";
+      }
+      leaf holddown {
```

Comment:
Any reason to not put the "holddown" leaf under an "ldp-sync" presence container like done for IS-IS instances? That would naturally prevent the user from configuring it unless LDP sync is configured first.

Reply:
I'm not sure how to implement the presence ldp-sync container just for  holddown.    I got a duplicate container.
2020/09/06 11:50:35 libyang: Duplicated container identifier "ldp-sync". (/frr-isisd:ldp-sync)

Reply:
We decided to make the config follow these rules: 

1. you must configure ldp-sync at the router isis/ospf level for ldp-sync to run
2. you can configure at interface level but that info is stored until ldp-sync
    is enabled at router level.  Interface config will continue to take precedence.
    But the interface config will not be removed if the user deletes mpls ldp-sync at the router level. 
4.  a new log is displayed if ldp-sync is not configured at router level:
    LDP-sync is disabled